### ISSUE TITLE
bl_test_image_io test is malformed

### ISSUE THREAD

**Issue by Fat-Zer** [blend2d/blend2d#217]
The test `bl_test_image_io` fails with the message:

```
$ ctest -R bl_test_image_io
Test project /tmp/blend2d/build
    Start 5: bl_test_image_io
1/1 Test #5: bl_test_image_io .................***Failed    0.00 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) =   0.01 sec

The following tests FAILED:
          5 - bl_test_image_io (Failed)
Errors while running CTest
Output from these tests are in: /tmp/blend2d/build/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
$ cat /tmp/blend2d/build/Testing/Temporary/LastTest.log
Start testing: Nov 08 08:58 GMT
----------------------------------------------------------
5/5 Testing: bl_test_image_io
5/5 Test: bl_test_image_io
Command: "/tmp/blend2d/build/bl_test_image_io"
Directory: /tmp/blend2d/build
"bl_test_image_io" start time: Nov 08 08:58 GMT
Output:
----------------------------------------------------------
Blend2D Image Codecs Tester [use --help for command line options]
  Version    : 0.11.4
  Build Type : Debug
  Compiled By: GCC 13.3

Failed to process command line arguments:
  A file to decode must be specified, use --file=<path> to specify it
<end of output>
Test time =   0.00 sec
----------------------------------------------------------
Test Failed.
"bl_test_image_io" end time: Nov 08 08:58 GMT
"bl_test_image_io" time elapsed: 00:00:00
----------------------------------------------------------

End testing: Nov 08 08:58 GMT
```

Obviously, it expects some `--file` to be passed but it is not specified in cmake

**Comment by kobalicek** [comment]
It's not malformed.

This test is a preparation for a much larger test suite designed to test image-io functionality, but the blend2d repository is not going to provide testing images - it will be another repository to make the main repository light.

Running `ctest` is unfortunately not enough to properly test the library, this file should provide an insight into what the CI is doing:

https://github.com/blend2d/blend2d/blob/master/.github/workflows/build-config.json

Basically running some tests multiple times with various parameters, etc...

**Comment by Fat-Zer** [comment]
Would you mind not to add `bl_test_image_io` to the tests via `add_test()` than?

> Running ctest is unfortunately not enough to properly test the library, this file should provide an insight into what the CI is doing:

It seems nice to add all this logic to cmake's test suite.

I can do both myself if you short on time and don't have anything against the idea.

**Comment by Fat-Zer** [comment]
Yep, everything seems to work fine, thanks.
